### ISSUE TITLE
Unable to get it running on 0.92.2.

### ISSUE THREAD

**Issue by GrumpyMeow** [tenly2000/HomeAssistant-Places#5]
Can anybody confirm if this places plugin still works?
I've put a few hours into it, but unable to get it working :-(

**Comment by emouawad** [comment]
Having issues too - It's related to 0.92.2

**Comment by knobunc** [comment]
Per https://community.home-assistant.io/t/reverse-geocode-sensor-places-using-openstreetmap-custom-component/52230/77 it looks like you need to add:

`{
  "domain": "Sensor",
  "name": "Reverse Geocode Sensor",
  "documentation": "https://github.com/tenly2000/HomeAssistant-Places",
  "requirements": [],
  "dependencies": ["zone","device_tracker"],
  "codeowners": []
}`

And possibly create an empty __init__.py file.

**Comment by emouawad** [comment]
Yeh that solves it - thanks

**Comment by GrumpyMeow** [comment]
I finally got it working.. I probably missed some instructions.
But:
* I already had the manifest.json file present as given by @knobunc 
* I hadn't provided the "api_key" parameter as i saw this was an optional config parameter. But eventually i read that the api_key had to be set to the emailadres of the account. Thus i also created an account at openstreetmap.org.
* I think i had to rename places.py to sensor.py in this path: /custom_components/places/sensor.py
* I added the __init__.py with some content:, so that it was not 0 bytes long: """coments"""

Many thanks. Great plugin.
